src/api.ts

```diff
@@ -29,6 +29,7 @@ export async function installDependencies(
       yarn: ["install", "--immutable"],
       bun: ["install", "--frozen-lockfile"],
       pnpm: ["install", "--frozen-lockfile"],
+      deno: ["install", "--frozen"],
```

Comment:
Since Deno [doesn't run lifecycle scripts](https://docs.deno.com/runtime/reference/cli/install/#options-allow-scripts) by default, should we include the `--allow-scripts` flag in the command?

Reply:
Mhmm i genuinely don't know what's the best option here @pi0 ?

Reply:
That's a good point however as per docs: 

> Note: Scripts will only be executed when using a node_modules directory (--node-modules-dir).

So I'm not sure either. I would say let's iterate and see if we need it in practice.